Bottom: 4f5a6881a69edae4d3e1c0891fc67f08096f7714
Top:    3aeab40fb32c4da5848193b3312d890f0136913d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-26 14:55:58 +0100

Refresh of add-stepcode-based-assembly

---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 56c5f8c..a228dc4 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -881,6 +881,8 @@ step_do_export (HID_Attr_Val * options)
           continue;
         model_filename = attribute;
 
+#if 0   /* Rather than write a parser for three floats in a string, separate X, Y, Z explicitly for quicker testing */
+
         attribute = AttributeGet (element, "PCB::3d_model::origin");
         if (attribute == NULL)
           continue;
@@ -897,7 +899,9 @@ step_do_export (HID_Attr_Val * options)
           continue;
         parse_direction_3d_string (attribute, &rx, &ry, &rz);
         rx = 1.0, ry = 0.0, rz = 0.0;
+#endif
 
+        /* XXX: Should parse a unit suffix, e.g. "degrees" */
         attribute = AttributeGet (element, "PCB::rotation");
         if (attribute == NULL)
           continue;
